example/lib/main.dart

```diff
@@ -40,7 +40,7 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(widget.title),
+        title: Text(widget.title!),
```

Comment:
Maybe can use `??` instead of force unwrapping
```suggestion
        title: Text(widget.title ?? ''),
```